lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -76,7 +81,7 @@ namespace {
 const std::string kPullParentPathKey("Maya:Pull:ParentPath");
 
 // Set name that will be used to hold all pulled objects
-const std::string kPullSetName("pullStateSet");
```

Comment:
The API being called using the name is taking a MString, so it made sense to keep the constant with the same type to avoid unnecessary copies when using it.